content/developer/howtos/javascript_fields.rst

```diff
@@ -0,0 +1,118 @@
+======
```

Comment:
split this into 2 how to: one titled "Create a new field component" and another one titled "Subclass an existing field component"

Reply:
An alternative could be to keep both sections on the same page but relabel the document to something like "Customize the display of a field", which provides two ways of doing so depending on your use case. Same thing for the views.